### ISSUE TITLE
Pixel 3a, GrapheneOS

### ISSUE THREAD

**Issue by r0ckr0ll** [aeri/Nihonoari-App#43]
Hi, the app does not "boot" properly anymore. It does not load past the opening screen.

Deinstalled and reinstalled via F-Droid.

**Comment by aeri** [comment]
Hi, thanks for the notice, it's also reported in #42 but we don't have more information about the bug.

Could you capture with ADB the error when starting the application?

It would also be great if you could test this version that loads very few components and resources to rule out possible exterbal packages.

[builds-apk.zip](https://github.com/aeri/Nihonoari-App/files/11142932/builds-apk.zip)

**Comment by r0ckr0ll** [comment]
I got the "untitled" / flutter app installed. It's basically a cookie clicker? It's working, running fine.

ADB: 

I'm having trouble getting adb to run on my computer. I can't get Android Studio [a] installed properly and the seperate platform tools I've found here [b] don't run, too.

[a] https://developer.android.com/studio

[b] https://www.androidpolice.com/2017/01/05/google-makes-adb-fastboot-platform-tools-available-without-full-sdk-android-studio-download/

Is this supposed to be run on Linux? If so I can switch to a virtual machine. I'll try that later. I'm using Win10.

Would a command like "adb logcat > file.txt" result in the output you need?

**Comment by aeri** [comment]
Thank you very much for your help and cooperation in trying to solve this problem

>I got the "untitled" / flutter app installed. It's basically a cookie clicker? It's working, running fine.

Great, that basic Flutter app works fine, have you been able to try the other one that goes in the .zip? The one called ```nihonoari-min```

This is Nihonoari but with much less stuff, in fact it is not functional, it will be a matter of trying to add more functionality and see where it fails.

>Is this supposed to be run on Linux? If so I can switch to a virtual machine. I'll try that later. I'm using Win10.

ADB logcat should work correctly on Windows 10, in principle it is not necessary to install the entire Android Studio suite to use the platform tools, Android provides them separately in a .zip file. 

It can be downloaded from here: https://developer.android.com/studio/releases/platform-tools?hl=es-419

>Would a command like "adb logcat > file.txt" result in the output you need?

Indeed, with this command before opening the application we will be able to see the trace of the system and the app to check if any type of error or exception occurs.

**Comment by r0ckr0ll** [comment]
> Great, that basic Flutter app works fine, have you been able to try the other one that goes in the .zip? The one called `nihonoari-min`

Both of them installed as the Flutter app. Yesterday.
I've checked again. The minimized version does not boot, too.

> It can be downloaded from here: https://developer.android.com/studio/releases/platform-tools?hl=es-419

This is the same file as in 
https://www.androidpolice.com/2017/01/05/google-makes-adb-fastboot-platform-tools-available-without-full-sdk-android-studio-download/
What's weird is that it wasn't available on developer.android.com if the language is set to English.

**Comment by r0ckr0ll** [comment]
[2023-04-04_Nihonoari_Pixel_3a_GrapheneOS.txt](https://github.com/aeri/Nihonoari-App/files/11149328/2023-04-04_Nihonoari_Pixel_3a_GrapheneOS.txt)

I've tested the app, uninstalled it, installed the official version, tested that one, deinstalled that one, reinstalled the minimized one. That's all covered in the log. I hope that'll help, if you need further data, tell me how to provide you with that.

**Comment by aeri** [comment]
Very good news, I have found the bug thanks to your trace that has been enlightening.

The point has been this error that has been repeated on all 3 occasions:

```
Unhandled Exception: Unable to load asset: i18n/de.json
```
I have noticed that I am indeed missing to add in the ```pubspec.yaml```

https://github.com/aeri/Nihonoari-App/blob/f19bf9d4ffba6c9145af825e1f32f91be8d24345/pubspec.yaml#L48-L57

It was difficult to find the bug because it only occurred in the German language, I will upload the update today to fix this critical bug.

Thank you very much for your collaboration and willingness, it has been fundamental.
